### PR TITLE
feat(Vtiger): Create Record Action / Delete Record  Action / Get Record Action / Make Api Call  Action/ Search Record Action / Update Record Action / New or Updated Records

### PR DESCRIPTION
**Pull request by kanarelo**
## What does this PR do?

This PR introduces the piece for vtiger. 

VTiger is an opensource php based CRM, with community and enterprise versions. Enterprise version has a new and updated API, but the community version has a slighly outdated API.

This Piece attempts to bring over features as seen on make.com.

https://www.make.com/en/integrations/vtiger

Commisioned by George - https://community.activepieces.com/t/vtiger-self-hosted-pieces/248/9

ANNOUNCEMENT=true

### CODE REVIEW

File: packages/pieces/vtiger/src/lib/actions/search2.html
Comment (abuaboud) [inline packages/pieces/vtiger/src/lib/actions/search2.html:1]: Is this used? If not can we remove it please

File: packages/backend/.env
Comment (abuaboud) [inline packages/backend/.env:1]: can you please rollback these changes

**Review by abuaboud — CHANGES_REQUESTED** [review]

File: packages/backend/.env
Comment (kanarelo) [reply packages/backend/.env:1]: sure

File: packages/pieces/vtiger/src/lib/actions/search2.html
Comment (kanarelo) [reply packages/pieces/vtiger/src/lib/actions/search2.html:1]: Yes, I think I added that by mistake

Comment (kanarelo) [reply packages/pieces/vtiger/src/lib/actions/search2.html:1]: fixed

File: packages/backend/.env
Comment (kanarelo) [reply packages/backend/.env:1]: fixed, check and let me know

**Review by abuaboud — APPROVED** [review]

**Review by abuaboud — APPROVED** [review]

**Review by abuaboud — APPROVED** [review]

**Pull request merged**